modules/selenium/src/test/java/org/testcontainers/junit/BrowserWebDriverContainerTest.java

```diff
@@ -51,34 +47,50 @@ public void provideDefaultNoProxyEnvironmentIfNotSet() {
     public void createContainerWithShmVolume() {
         try (
             BrowserWebDriverContainer webDriverContainer = new BrowserWebDriverContainer()
+                .withCapabilities(new FirefoxOptions())
         ) {
             webDriverContainer.start();
 
-            final List<InspectContainerResponse.Mount> mounts = webDriverContainer.getContainerInfo().getMounts();
-            assertEquals("Shm mounts present", mounts.size(), 1);
-
-            final InspectContainerResponse.Mount shmMount = mounts.get(0);
-            assertEquals("Shm mount source is correct", "/dev/shm", shmMount.getSource());
-            assertEquals("Shm mount destination is correct", "/dev/shm", shmMount.getDestination().getPath());
-            assertEquals("Shm mount mode is correct", shmMount.getMode(), "rw");
+            assertEquals("Only one shm mount present", countCorrectlyConfiguredShmVolumes(webDriverContainer), 1L);
```

Comment:
if the assertion fails, one would have to run the test and debug why.

WDYT about replacing it with assert that expects the count of **any** SHM mount to be 1 and then comparing the source & target?

So that if it is 1, but source != target, we will see it immediately from the error

Reply:
... and the r/w mount type can be verified separately as well :)

Reply:
😆 doing something a bit like this, the entire thing collapses into something simpler and eliminates the need for both `countCorrectlyConfiguredShmVolumes` _and_ streams.